python-bindings/chiapos.cpp

```diff
@@ -68,12 +69,26 @@ PYBIND11_MODULE(chiapos, m)
                                       num_buckets,
                                       stripe_size,
                                       num_threads,
-                                      nobitfield);
+                                      nobitfield,
+                                      verbose);
                 } catch (const std::exception &e) {
                     std::cout << "Caught plotting error: " << e.what() << std::endl;
                     throw e;
                 }
-            });
+            },
+            py::arg("tmp_dir"),
+            py::arg("tmp2_dir"),
+            py::arg("final_dir"),
+            py::arg("filename"),
+            py::arg("k"),
+            py::arg("memo"),
+            py::arg("id"),
+            py::arg("buffmegabytes"),
+            py::arg("num_buckets"),
+            py::arg("stripe_size"),
+            py::arg("num_threads"),
+            py::arg("nobitfield"),
+            py::arg("verbose") = true);
```

Comment:
is there a good reason to enable verbose by default?

Reply:
Yes, the reason is "backward compatibility", the current behavior is to show the lines with "Block X uniform sort" text. this `verbose = false` is for avoid printing that. If we don't care about preserve the old behavior, I'm happy to set it to false by default.